### PR TITLE
charts: adapt for notebook server options

### PR DESCRIPTION
**Pull request by rokroskar**
compatibility with
https://github.com/SwissDataScienceCenter/renku-notebooks/pull/68

### CODE REVIEW

File: charts/renku/values.yaml [hunk 1]
```diff
@@ -60,10 +60,10 @@ ingress:
   ## Annotations for the created ingress
   annotations:
     ## The ingress class
-    kubernetes.io/ingress.class: 'nginx'
+    kubernetes.io/ingress.class: nginx
     # kubernetes.io/tls-acme: 'true'
     nginx.ingress.kubernetes.io/proxy-body-size: '0' # Adjust to a reasonable value for production to avoid DOS attacks.
-    nginx.ingress.kubernetes.io/proxy-request-buffering: 'off' # Needed if GitLab is behind this ingress
+    nginx.ingress.kubernetes.io/proxy-request-buffering: off   # Needed if GitLab is behind this ingress
```
Comment (leafty) [inline charts/renku/values.yaml]: :/

Comment (rokroskar) [reply charts/renku/values.yaml]: Aaaaaaah!

Comment (leafty) [reply charts/renku/values.yaml]: But the problem remains that chartpress will mess up at publish time -_-'

Comment (rokroskar) [reply charts/renku/values.yaml]: but the charts we are using now have been published the same way and deployed without errors...

**Comment by rokroskar** [comment]
continued as #451

**Pull request closed without merging**